### ISSUE TITLE
Suggestion: make the ":1" part of the ratio also big/bold

### ISSUE THREAD

**Issue by patrickhlauke** [WebAccessibilityTools/CCA#12]
A design/subjective/taste choice, but having the ":1" part of the contrast ratio small/deemphasised looks a bit odd

> ![screenshot showing 'contrast ratio 2.3:1' where the 2.3 part is large and emphasised](https://github.com/user-attachments/assets/c8c45522-8d78-4f47-9a6d-0f7b4f410307)

It may be worth making the ":1" part at least as big/bold as the contrast ratio number itself (and keeping it gray rather than black)
